### PR TITLE
[add] affinity field to istiod

### PR DESCRIPTION
**Pull request by ymktmk**
**Please provide a description of this PR:**

This is an addition to helm Chart's value.yaml !
Even though the affinity field is actually provided, there was no field of it in values.yaml, so I added it.

https://github.com/istio/istio/blob/master/manifests/charts/istio-control/istio-discovery/templates/deployment.yaml#L65~L68

### CODE REVIEW

**Comment by jacob-delgado** [comment]
This LGTM. Good find. I'll approve once the integration jobs pass. You may need to modify a the operator_values.proto file for this to work. Then do a `make gen`.

File: manifests/charts/istio-control/istio-discovery/values.yaml [hunk 1]
```diff
@@ -32,6 +32,8 @@ pilot:
 
   env: {}
 
+  affinity: {}
```
Comment (hanxiaop) [inline manifests/charts/istio-control/istio-discovery/values.yaml:35]: The operator api is in operator/pkg/apis/istio/v1alpha1/values_types.proto, you also have to modify it.

Comment (ymktmk) [reply manifests/charts/istio-control/istio-discovery/values.yaml:35]: thank you ! 
I modify it !

Comment (hanxiaop) [reply manifests/charts/istio-control/istio-discovery/values.yaml:35]: Looks like there's a conflict issue in the proto generated file, could you run `make gen` again and include the changes?

**Pull request closed without merging**